src/utils/getTokenPath.ts

```diff
@@ -0,0 +1,43 @@
+import { Protocol } from '@uniswap/router-sdk'
```

Comment:
i think a utils folder under swap would be more meaningful - i think the path `src/utils` gives you no real information about what belongs in here

Reply:
Might be something to look at in the future - right now, we have all utilities in `src/utils`, such as `calculateSlippageAmount` that would be also very useful to be grouped as swap/token/trade related. Maybe we should consider having that form of grouping in the future?